corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -899,13 +940,19 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
             return {
                 "role": "link",
                 "tabindex": "0",
+                "style": this.buildMaxWidth(),
             };
         },
         events: {
             "click": "crumbClick",
             "keydown": "crumbKeyAction",
         },
-
+        buildMaxWidth: function () {
+            // to avoid overflow, compute the max width in CSS based on number of breadcrumbs
+            const fullWidthOffset = 106.5; // padding including home and hamburger icons
```

Comment:
It could change if we modified/removed the hamburger menu or home icons, yes. I think ideally we would actively identify the available space instead of just assuming it's "everything minus a bit" but that's surprisingly tough to do. As-is, I think you're right it makes sense to put next to the other breadcrumb-related constant. https://github.com/dimagi/commcare-hq/pull/33453/commits/92bb5d926a9664700293329effab147f5a1f1d7f